main/src/library/RedBlackTree.flix

```diff
@@ -707,41 +707,13 @@ namespace RedBlackTree {
     /// Note that this is equivalent to `seqMinimumBy` but for performance reasons -- to avoid
     /// megamorphic calls -- we use a copy here.
     ///
-    def parSeqMinimumBy(cmp: (k, v, k, v) -> Comparison & ef, t: RedBlackTree[k, v]): Option[(k, v)] & ef = match t {
```

Comment:
Sorry. I am tired. Explaining badly. I want to keep parSeqMinimumBy (due to inlining, ill explain later). 

I only wanted to update seqMinimumBy to use reduceLeft